2022/Q4/artifacts/PoS-pseudocode/PoS-model.md

```diff
@@ -510,19 +524,21 @@ end_of_epoch()
 
   var set_validators = {val | val = slash.validator && slash in enqueued_slashes[cur_epoch]}
   forall (validator_address in set_validators) do
+    var total_staked = read_epoched_field(validators[validator_address].total_deltas, slash.epoch, 0)
```

Comment:
`slash` is undefined here.

Reply:
I've changed `end_of_epoch()` again now in 61c733f to correct what I had done before and I think to solve the issue of `total_unbonded > total_staked`